libs/authentication/website/src/lib/login-form/login-form.tsx

```diff
@@ -70,13 +80,17 @@ export function LoginForm({
       password: '',
       requirePassword: defaults?.requirePassword || false,
     },
-    mode: 'onTouched',
-    reValidateMode: 'onChange',
+    mode: 'onSubmit',
+    reValidateMode: 'onBlur',
   });
 
   const onSubmit = handleSubmit(({ email, requirePassword, password }) => {
     if (requirePassword) {
-      return onSubmitLoginWithCredentials(email, password);
+      setAwaitingWithCredentials(true);
+      return (async () => {
```

Comment:
Look up async action I think its called, comments should have it.

Helps you abstracting this better.